src/preload.js

```diff
@@ -53,6 +54,29 @@ import preCuePostBlock2 from "./assets/video/en/precue/postBlock2.mp4";
 import preCuePostBlock3 from "./assets/video/en/precue/postBlock3.mp4";
 import preCuePostBlock4 from "./assets/video/en/precue/postBlock4.mp4";
 import preCueEnd from "./assets/video/en/precue/end.mp4";
+// Generic visual attention versions (dots only)
+// TODO (Maha): If you rename or add video assets, be sure to import them in the lines below
+import genericIntro1 from "./assets/video/en/generic/intro1Generic.mp4";
+import genericIntro2 from "./assets/video/en/generic/intro2Generic.mp4";
+import genericIntro3 from "./assets/video/en/generic/intro3Generic.mp4";
+import genericIntro4 from "./assets/video/en/generic/intro4Generic.mp4";
+import genericPostPractice from "./assets/video/en/generic/postPracticeGeneric.mp4";
+import genericPostBlock1 from "./assets/video/en/generic/postBlock1Generic.mp4";
+import genericPostBlock2 from "./assets/video/en/generic/postBlock2Generic.mp4";
+import genericPostBlock3 from "./assets/video/en/generic/postBlock3Generic.mp4";
+import genericPostBlock4 from "./assets/video/en/generic/postBlock4Generic.mp4";
+import genericEnd from "./assets/video/en/generic/endGeneric.mp4";
+// Spanish language versions
+import genericIntro1 from "./assets/video/es/generic/intro1Generic.mp4";
+import genericIntro2 from "./assets/video/es/generic/intro2Generic.mp4";
+import genericIntro3 from "./assets/video/es/generic/intro3Generic.mp4";
+import genericIntro4 from "./assets/video/es/generic/intro4Generic.mp4";
+import genericPostPractice from "./assets/video/es/generic/postPracticeGeneric.mp4";
+import genericPostBlock1 from "./assets/video/es/generic/postBlock1Generic.mp4";
+import genericPostBlock2 from "./assets/video/es/generic/postBlock2Generic.mp4";
+import genericPostBlock3 from "./assets/video/es/generic/postBlock3Generic.mp4";
+import genericPostBlock4 from "./assets/video/es/generic/postBlock4Generic.mp4";
+import genericEnd from "./assets/video/es/generic/endGeneric.mp4";
```

Comment:
The imported Spanish videos will need to have slightly different variable names than the English ones. Suggestion below

```suggestion
import genericIntro1Es from "./assets/video/es/generic/intro1Generic.mp4";
import genericIntro2Es from "./assets/video/es/generic/intro2Generic.mp4";
import genericIntro3Es from "./assets/video/es/generic/intro3Generic.mp4";
import genericIntro4Es from "./assets/video/es/generic/intro4Generic.mp4";
import genericPostPracticeEs from "./assets/video/es/generic/postPracticeGeneric.mp4";
import genericPostBlock1Es from "./assets/video/es/generic/postBlock1Generic.mp4";
import genericPostBlock2Es from "./assets/video/es/generic/postBlock2Generic.mp4";
import genericPostBlock3Es from "./assets/video/es/generic/postBlock3Generic.mp4";
import genericPostBlock4Es from "./assets/video/es/generic/postBlock4Generic.mp4";
import genericEndEs from "./assets/video/es/generic/endGeneric.mp4";
```